WordPress/twentytwentyfive
# News blog templates: Add main element

**Description**

Adds the `<main>` HTML element to the news blog templates to improve the accessibility.

Partial for 
https://github.com/WordPress/twentytwentyfive/issues/278

**Testing Instructions**
Test all of the news blog templates:
- There should be no visual change on the front of the site
- All templates should have one `<main>` element
- All templates should have a "skip link":
 Place the curser in the address bar of the browser. Press the `Tab` key: move past the WordPress admin bar.
 Confirm that there is a skip link:
 
![image](https://github.com/user-attachments/assets/105d8046-8df8-49c9-baa9-a7b6411ccea7)
